packages/kbn-securitysolution-grouping/src/hooks/use_get_group_selector.tsx

```diff
@@ -120,10 +122,15 @@ export const useGetGroupSelector = ({
         return;
       }
 
-      const newSelectedGroups = isNoneGroup([groupSelection])
-        ? [groupSelection]
-        : [...selectedGroups.filter((selectedGroup) => selectedGroup !== 'none'), groupSelection];
-      setSelectedGroups(newSelectedGroups);
+      // Simulate a toggle behavior when maxGroupingLevels is 1
```

Comment:
maybe move this to a variable to avoid the need for the comment, smth like `const shouldToggle = maxGroupingLevels === 1` or smth